diff --git a/CMakeLists.txt b/CMakeLists.txt
index c6150948..a043b71c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,8 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(cassandra C CXX)
 
+set(CMAKE_MACOSX_RPATH 1)
+
 set(CASS_ROOT_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(CASS_SRC_DIR "${CASS_ROOT_DIR}/src")
 set(CASS_INCLUDE_DIR "${CASS_ROOT_DIR}/include")
@@ -29,13 +31,13 @@ endif()
 
 option(CASS_BUILD_EXAMPLES "Build examples" OFF)
 option(CASS_BUILD_INTEGRATION_TESTS "Build integration tests" OFF)
-option(CASS_BUILD_SHARED "Build shared library" ON)
-option(CASS_BUILD_STATIC "Build static library" OFF)
+option(CASS_BUILD_SHARED "Build shared library" OFF)
+option(CASS_BUILD_STATIC "Build static library" ON)
 option(CASS_BUILD_TESTS "Build tests" OFF)
 option(CASS_BUILD_UNIT_TESTS "Build unit tests" OFF)
 option(CASS_DEBUG_CUSTOM_ALLOC "Debug custom allocator" OFF)
 option(CASS_INSTALL_HEADER "Install header file" ON)
-option(CASS_INSTALL_HEADER_IN_SUBDIR "Install header file under 'include/cassandra'" OFF)
+option(CASS_INSTALL_HEADER_IN_SUBDIR "Install header file under 'include/cassandra'" ON)
 option(CASS_INSTALL_PKG_CONFIG "Install pkg-config file(s)" ON)
 option(CASS_MULTICORE_COMPILATION "Enable multicore compilation" ON)
 option(CASS_USE_BOOST_ATOMIC "Use Boost atomics library" OFF)
